test/utils/generateBotToken.js

```diff
@@ -13,4 +13,12 @@ const generateToken = (data) => {
   });
 };
 
-module.exports = { generateToken };
+const generateCronJobToken = (data) => {
+  const token = jwt.sign(data, config.get("cronJobHandler.privateKey"), {
+    algorithm: "RS256",
+    expiresIn: "1m",
```

Comment:
seems like a magic number.
Can be stored as a constant with an appropriate name and a comment besides mentioning why exactly 1 min.
FYI https://en.wikipedia.org/wiki/Magic_number_(programming)

Reply:
Hey, @apurvkhare This is a test util, its a dummy auth token generator only for testing purposes, it is not a part of the build. Do we still need a config constant?

Our use-case just requires verifying auth token it will be generated from an external service.

Reply:
didn't notice that, its fine then